examples/hybrid/define_post_processing.jl

```diff
@@ -620,6 +621,49 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
             output_dir * "/bw-vert_intg_water_vapor-day" * string(day) * ".png",
         )
 
+        # Spectrum calculation
+        mass_weight = ones(FT, 1) # only 1 level
+        # First compute 1d spectrum of u-component
+        u_1d_spectrum, freqs = power_spectrum_1d(
+            FT,
+            u,
+            similar(mass_weight),
+            lat,
+            lon,
+            mass_weight,
+        )
+        # Then compute 2d spectrum of u-component
+        u_2d_spectrum, wave_numbers, spherical, mesh_info =
+            power_spectrum_2d(FT, u[:, :, 1, 1], mass_weight) # use first level, 1500m
+        # Now repeat with v-component, i.e., compute 1d spectrum of v-component
+        v_1d_spectrum, freqs = power_spectrum_1d(
+            FT,
+            v,
+            similar(mass_weight),
+            lat,
+            lon,
+            mass_weight,
+        )
+        # Then compute 2d spectrum of v-component
+        v_2d_spectrum, wave_numbers, spherical, mesh_info =
+            power_spectrum_2d(FT, v[:, :, 1, 1], mass_weight) # use first level, 1500m
+
+        spectrum_1d = 0.5 .* (u_1d_spectrum + v_1d_spectrum)
+        spectrum_2d = 0.5 .* (u_2d_spectrum + v_2d_spectrum)
+        plot_spectrum = Plots.contourf(
+            collect(0:1:(mesh_info.num_fourier))[:],
+            collect(0:1:(mesh_info.num_spherical))[:],
+            (spectrum_2d[:, :, 1])',
+            xlabel = "m",
+            ylabel = "n",
+            color = :balance,
+            title = "2d spectrum (1500m) day " * string(day),
```

Comment:
Does any vertical interpolation applied somewhere in the calculation? If not, in most of current set-ups, first layer may not correspond to 1500 meters.

Reply:
Oh, I had referred to lines 574, 576 above where something similar was done for `u`. Is that different?

Reply:
There might be a problem there as well ... This was originally hard-coded for the uniform grids of 10 layers with model top at 30km.

Reply:
If we want to fix all title strings, then I'd propose doing it in a different clean-up PR.